EIPS/eip-2935.md

```diff
@@ -24,38 +24,31 @@ A side benefit of this approach could be that it allows building/validating proo
 
 ## Specification
 
-| Parameter | Value |
-| - | - |
-| `FORK_TIMESTAMP` | TBD |
-| `HISTORY_STORAGE_ADDRESS` | `0x25a219378dad9b3503c8268c9ca836a52427a4fb`|
-| `HISTORY_SERVE_WINDOW` | `8192` |
-| `BLOCKHASH_OLD_WINDOW` | `256` |
+| Parameter                 | Value                                        |
+| ------------------------- | -------------------------------------------- |
+| `FORK_TIMESTAMP`          | TBD                                          |
+| `VERKLE_FORK_TIMESTAMP`   | TBD                                          |
+| `HISTORY_STORAGE_ADDRESS` | `0x25a219378dad9b3503c8268c9ca836a52427a4fb` |
+| `HISTORY_SERVE_WINDOW`    | `8192`                                       |
+| `BLOCKHASH_OLD_WINDOW`    | `256`                                        |
```

Comment:
```suggestion
| `BLOCKHASH_SERVE_WINDOW`    | `256`                                        |
```
I think this name would be better since we wouldn't modify blockhash window even when we serve it from state